prometheus-exporters/openstack-exporter/values.yaml

```diff
@@ -1,4 +1,6 @@
 owner-info:
+  support-group: compute-storage-api
+  service: cinder
```

Comment:
I'd leave out the service, because it's a generic "openstack" exporter - currently only used by cinder - unless you feel like only cinder-folk should maintain it.

Also one of the maintainers is not in the appropriate support_group.

Reply:
done